### PR TITLE
 high availability across multiple datacenter #2587 

### PR DESCRIPTION
**Pull request by voriol**
documentations about how deploy a high availability across multiple datacenters #2587

### CODE REVIEW

**Review by CyberDem0n — COMMENTED** [review]
Thank you @voriol!

Please try to avoid using "master" (or "M" on diagrams).
About the first one I already suggested changes, the latter should still be addressed.

**Comment by voriol** [comment]
Hi @CyberDem0n, sorry for using the "master" annotation, I have corrected the graph and accepted your suggestions.

Thank you!

File: docs/ha_multi_dc.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+.. image:: _static/multi-dc-synchronous-replication.png
+
+We must deploy a cluster of etcd, ZooKeeper or Consul through the different DC, with a minimum of 3 nodes, one in each zone.
+
+Regarding postgres, we must deploy at least 2 nodes, in different DC. Then you have to set ``synchronous_mode: true`` in the global configuration (``patronictl edit-config``).
+
+This enables sync replication and the primary node will choose one of the nodes as synchronous.
+
+Streaming Replication (asynchronous)
```
Comment (hughcapet) [inline docs/ha_multi_dc.rst:29]: From this title it looks like only async replication is streaming. Can we just have 'Asynchronous Replication' here?

Comment (voriol) [reply docs/ha_multi_dc.rst]: for sure! i have already corrected the title, thanks for the feedback

**Comment by CyberDem0n** [comment]
:+1:

**Comment by CyberDem0n** [comment]
Merged, thank you @voriol!

**Pull request merged**